kalite/distributed/management/commands/points_reset.py

```diff
@@ -0,0 +1,18 @@
+"""
+"""
+from django.core.management.base import BaseCommand, CommandError
+from kalite.main.models import ExerciseLog, VideoLog, AttemptLog
+
+class Command(BaseCommand):
+    help = "Reset Points to zero for all the student accounts"
+
+    def handle(self, *args, **options):
+    	for entry in ExerciseLog.objects.all():
```

Comment:
Using [update](https://docs.djangoproject.com/en/dev/ref/models/querysets/#update) is a more efficient way to do this, if you don't need any special behaviour in `save`.